src/components/shapes/attributes.js

```diff
@@ -235,8 +235,30 @@ module.exports = templatedArray('shape', {
         role: 'info',
         editType: 'arraydraw',
         description: [
-            'Sets the color filling the shape\'s interior.'
+            'Sets the color filling the closed shape\'s interior.'
         ].join(' ')
     },
+    fillrule: {
+        valType: 'enumerated',
+        values: ['evenodd', 'nonzero'],
+        dflt: 'evenodd',
+        role: 'info',
+        editType: 'arraydraw',
+        description: [
+            'Determines the path\'s interior.',
```

Comment:
```suggestion
            'Determines which regions of complex paths constitute the interior.',
```

Reply:
Thanks for your suggestion @alexcjohnson. 
Resolved by https://github.com/plotly/plotly.js/pull/4775/commits/838bda91346430351e0f705d1a8f452693a6f6b5.